x-pack/test/observability_functional/apps/observability/pages/alerts/index.ts

```diff
@@ -73,7 +73,8 @@ export default ({ getService }: FtrProviderContext) => {
           await observability.alerts.common.submitQuery('');
         });
 
-        it('Autocompletion works', async () => {
+        // TODO: This test fails on CI but succeeds locally
```

Comment:
Haven't been able to diagnose, I'll open an issue

Reply:
Looking at other test failure issues and realizing I shouldn't open one till this merges, but then it would require another PR to add the issue URL to the comments

@elastic/obs-ux-management-team would y'all be able to help me figure out why this might be failing on CI but passing locally? Or if it's a more difficult problem we can figure out how to track the test failure somewhere